server/main.py

```diff
@@ -59,14 +59,19 @@ def interop_get(key):
 
 @app.route("/interop/telemetry")
 def interop_telemetry():
-    return jsonify(interop.telemetry_json)
+    return json.dumps(interop.telemetry_json)
```

Comment:
What exactly is the reason for using json.dumps over jsonify here (and in line 72)? I believe jsonify is henerally better, and comes with some extra features for Flask APIs.